data-prepper-plugins/avro-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/avro/AvroOutputCodec.java

```diff
@@ -179,6 +176,9 @@ private GenericRecord buildAvroRecord(final Schema schema, final Map<String, Obj
                 continue;
             }
             final Schema.Field field = schema.getField(key);
+            if(field == null) {
```

Comment:
(Only because you fixed other spacing issues in this PR)

You are missing a space after the `if`

```
if (field == null) {
```